rootfs/opt/fluentd/sbin/boot

```diff
@@ -1,6 +1,18 @@
 #!/bin/bash
 FLUENTD_CONF="/opt/fluentd/conf/fluentd.conf"
 
+if [ $INSTALL_BUILD_TOOLS == "true" ]
+then
+  apt-get update
+  apt-get install -y \
+    g++ \
```

Comment:
I'd just put those into a variable and reference it further down during the remove